source/libs/on-pr-merge-panel-open.ts

```diff
@@ -0,0 +1,39 @@
+// Memoization here is used to let onPrMergePanelOpen() be called multiple times without risking multiple attached handlers
+import mem from 'mem';
+import delegate, {DelegateSubscription, DelegateEvent} from 'delegate-it';
+
+const delegateHandler = mem((callback: EventListener) => (event: DelegateEvent) => {
+	if (event.delegateTarget.matches('.open')) {
+		callback(event);
+	}
+});
+
```

Comment:
I love this pattern.

Since `addEventListener` doesn't add the same handler twice, it's important that the same exact handler is passed every time. To do this I always use a `WeakMap` and an ugly sequence of `get or create, then set and return`, but I think I finally found a way to hide this complexity with `mem`.

Reply:
I like this pattern a lot 👍

Reply:
Also used here https://github.com/bfred-it/delegate-it/pull/10

<img width="289" alt="Screenshot 2019-05-17 at 21 58 06" src="https://user-images.githubusercontent.com/1402241/57933071-e02b6280-78ee-11e9-9834-799896ff5a47.png">